src/frontend/packages/lib_components/src/common/ErrorComponents/index.tsx

```diff
@@ -175,6 +176,22 @@ const messages = {
       id: 'components.ErrorComponents.videoDeleted.title',
     },
   },
+  fileTooLarge: {
+    text: {
+      defaultMessage: `This file is too large to be uploaded. Max size allowed is :
+      - 1MB for subtitles
+      - 10MB for images
+      - 300MB for shared documents
+      - 1GB for videos`,
```

Comment:
As this is a setting in the backend, it is subject to change and in production this values are changed. So we can't so it like this.
As suggested once, this size should be added, for each resource, in the api metada. I can show you how to do that.